### PR TITLE
scientific-consensus: guard five untested edges of the controversies command

### PR DESCRIPTION
**Pull request by laci141**
…command

The command's existing tests cover the four-stance tally, the pinned score and the printed breakdown. Five other decisions in the same 149 lines had no coverage at all, each one a silent-failure path.

Adds internal/cli/controversies_edges_test.go, hand-authored beside the generated controversies_test.go, with one test per decision. All five are mutation-verified: breaking the guarded line fails that test and only that test.

  contested threshold   dropping "&& directional >= 4" lets a 1/1 split
                        report CONTESTED on two works
  note ordering         swapping the two branches makes an empty corpus say
                        the evidence is thin rather than absent; with
                        studyCount 0 both conditions hold, so only the order
                        decides
  zero directional      removing "if directional > 0" does NOT surface as a
                        NaN — round2f converts through int, and NaN to int is
                        undefined in Go, so the measured result was
                        -9.223372036854776e+16, a finite number that marshals
                        to valid JSON. Worse than a NaN, which encoding/json
                        rejects outright. The test asserts the value, not
                        just IsNaN
  rounding              dropping the +0.5 in round2f turns 2*4/9 = 0.888...
                        into 0.88 instead of 0.89
  empty sides           removing either len()>0 guard prints a "Supporting
                        side:" heading with nothing under it

No .printing-press-patches record: not one line of controversies.go changes, so there is no hand edit for a regeneration to overwrite. Every existing record in this directory lists a code file beside its test.

## Summary

- 

## Published CLI Metadata

Use this section for Printing Press library publishes. Delete it only when the
PR does not add or modify `library/<category>/<slug>/`.

**API:** `<slug>` | **Category:** `<category>` | **Press version:** `<version>`
**Spec:** `<spec_url or spec.yaml>`  
**Required env:** `<none or env vars>`

## What Changed

- 

## CLI Shape

```bash
$ <cli-name> --help
<paste high-signal help output>
```

## What This CLI Does

<Use the first 2-3 README paragraphs or a concise equivalent.>

## Manuscripts

- [Research Brief](./library/<category>/<slug>/.manuscripts/<run-id>/research/)
- [Shipcheck Results](./library/<category>/<slug>/.manuscripts/<run-id>/proofs/)

## Validation
- [ ] `python3 .github/scripts/verify-skill/verify_skill.py --dir library/<category>/<slug>/`
- [x] `go build ./...` from the touched CLI root
- [x] `go vet ./...` from the touched CLI root
- [x] `go test ./...` from the touched CLI root
- [x] `govulncheck ./...` from the touched CLI root
- [ ] `go run ./tools/generate-skills/main.go` when `library/**/SKILL.md` changed
- [x] `git diff --check`

The two unchecked boxes are not applicable to this PR: `verify_skill.py` is the
publish-flow check for new-CLI submissions, and `generate-skills` only runs when
a `SKILL.md` changes. This PR adds a test file and a patch record; no `SKILL.md`
is touched.

## Gaps / Follow-Up

- ### Why no .printing-press-patches record

Not one line of controversies.go changes — this PR adds a test file and
nothing else, so there is no hand edit for a regeneration to overwrite.
All eighteen records in this directory list a code file beside their test;
the record's subject is the code change, with the test as its guard.

### Why a new file rather than controversies_test.go

controversies_test.go carries the "DO NOT EDIT" generated header, so tests
written into it would vanish silently on the next regeneration. Same pattern
as retraction_test.go in #1948.

### CODE REVIEW

File: library/other/scientific-consensus/internal/cli/controversies_edges_test.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
+// Hand-authored edge-case tests for the controversies command. Not generated.
```
[the remark this answers is not in the document]
Comment (laci141) [reply library/other/scientific-consensus/internal/cli/controversies_edges_test.go:1]: You're right, and the PR body's reasoning was wrong. I read
"code-level customizations" as excluding tests, but AGENTS.md scopes the
convention to anything beyond what the generator produced, and the litmus
test in the same section is decisive: a regeneration would not bring this
file back.

Added controversies-edge-decisions-are-guarded.json in 809b1e1. It records
the five branches that must survive a print, not the assertions, and notes
why the file sits beside the generated controversies_test.go rather than
inside it.
